apps/issuance/src/issuance.service.ts

```diff
@@ -1366,7 +1366,9 @@ async sendEmailForCredentialOffer(sendEmailCredentialOffer: SendEmailCredentialO
           platformName: bulkPayloadDetails?.platformName,
           certificate: bulkPayloadDetails?.certificate,
           size: bulkPayloadDetails?.size,
-          orientation: bulkPayloadDetails?.orientation
+          orientation: bulkPayloadDetails?.orientation,
+          height: bulkPayloadDetails?.height,
```

Comment:
`bulkPayloadDetails` already destructured at the starting of the function, you can use the value in similar way